src/main/java/com/benko/exercise/sort/SortingApp.java

```diff
@@ -0,0 +1,17 @@
+package com.benko.exercise.sort;
+
+public class SortingApp {
+
+    public static void main(String[] args) {
+        BubbleSorter bubbleSorter = new BubbleSorter();
+        int[] numbers = new int[]{4, 2, 0, 2, 10, 1, 9, 12, 3, 5};
+        bubbleSorter.sort(numbers);
+        SortingApp.printNumbers(numbers);
```

Comment:
Instead of `SortingApp.printNumbers(numbers);` please use just `printNumbers(numbers);`
Both methods (main and printNumbers) are in the same class so there's no need to reference method by class name.

Reply:
done